phone-store/e2e/protractor.conf.js

```diff
@@ -15,7 +15,7 @@ exports.config = {
   capabilities: {
     browserName: 'chrome'
   },
-  directConnect: true,
+  directConnect: false,
```

Comment:
I'm not sure why you changed this from the default. I can see why you might want to from what I've read on-line, but then I'm not sure why they have it set to `true` in the default config?

Reply:
We set this and moved to going through selenium in lab 3, 4, and the iteration template so we could use the element highlighting built into protractor.

Reply:
So this is necessary to make the highlighting work? Would it run (appreciably) faster if we _didn't_ have that?

I'm not convinced that students use the highlighting much, especially once there are a lot of tests, although @kklamberty probably has differing opinions on that.

Reply:
If it's not needed then we could remove it and also possibly change back a lot of the custom stuff we did to allow for the highlighting but we would want to do a good amount of testing, especially on lab machines, to make sure it still works correctly with the more default setup.